test/Microsoft.ML.Functional.Tests/Validation.cs

```diff
@@ -12,6 +12,7 @@
 using Microsoft.ML.Trainers.LightGbm;
 using Xunit;
 using Xunit.Abstractions;
+using static Microsoft.ML.TrainCatalogBase;
```

Comment:
What is this used for?

Reply:
It's to call the `mlContext.Regression.CrossValidate` function on line 160.